verifier/src/qe_identity.rs

```diff
@@ -217,37 +237,25 @@ impl<'a> SignedQeIdentity<'a> {
     ///   or equivalent
     pub fn verify(self, key: &VerifyingKey, time: DateTime) -> Result<(), Error> {
         self.verify_signature(key)?;
-        self.verify_time(time)?;
+        let qe_identity = QeIdentity::try_from(&self)?;
+        qe_identity.verify_version()?.verify_time(time)?;
```

Comment:
Maybe there should just be a `.verify()` that does both of these?

Reply:
Good idea, that provides symmetry with the verify on `SignedQeIdentity`
Done.